backend/src/main/java/io/debezium/configserver/rest/ConnectorResource.java

```diff
@@ -447,4 +447,49 @@ public Response deleteConnector(
         }
         return deleteResponse;
     }
+
+    @Path(ConnectorURIs.CONNECTOR_CONFIG_ENDPOINT)
+    @GET
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    @APIResponse(
+            responseCode = "204",
+            content = @Content(mediaType = MediaType.APPLICATION_JSON))
+    @APIResponse(
+            responseCode = "404",
+            description = "Connector with specified name not found")
+    @APIResponse(
+            responseCode = "500",
+            description = "Exception during Kafka Connect URI validation",
+            content = @Content(
+                    mediaType = MediaType.APPLICATION_JSON,
+                    schema = @Schema(implementation = ServerError.class)
+            ))
+    @APIResponse(
+            responseCode = "503",
+            description = "Exception while trying to connect to the selected Kafka Connect cluster",
+            content = @Content(
+                    mediaType = MediaType.APPLICATION_JSON,
+                    schema = @Schema(implementation = ServerError.class)
+            ))
+    public Response getConnectorConfig(
+            @PathParam("cluster") int cluster,
+            @PathParam("connectorname") String connectorName
+    ) throws KafkaConnectClientException, KafkaConnectException {
+        URI kafkaConnectURI = KafkaConnectClientFactory.getKafkaConnectURIforCluster(cluster);
+        KafkaConnectClient kafkaConnectClient = KafkaConnectClientFactory.getClient(cluster);
+
+        Map<String, String> connectorConfig = Collections.emptyMap();
+        try {
+            connectorConfig = kafkaConnectClient.getConnectorConfig(connectorName);
```

Comment:
Could you please check the response headers of that response? I think you have to create a response copy with `Response.from()` like in other methods here to set a proper response content-type.

Reply:
I think the doc [here](https://docs.confluent.io/platform/current/connect/references/restapi.html#get--connectors-(string-name)-config) says the response is a map?  If not, can you point me to an example to fix?

Reply:
Just search for `Response.from` in this class. We have some other proxy methods that forward the original response and re-append the proper content-type.